infra/core/gateway/api-management.bicep

```diff
@@ -19,8 +19,8 @@ param publisherEmail string = 'noreply@microsoft.com'
 param publisherName string = 'n/a'
 
 @description('The pricing tier of this API Management service')
-@allowed([ 'Consumption', 'Developer', 'Standard', 'Premium'])
-param sku string = 'Consumption'
+@allowed([ 'Consumption', 'Developer', 'Standard', 'Premium', 'BasicV2', 'StandardV2'])
+param sku string = 'StandardV2'
```

Comment:
I would prefer to see BasicV2 be the default here - it keeps cost down for a sample.

Reply:
Done